crates/stdarch-gen-arm/spec/neon/aarch64.spec.yml

```diff
@@ -13023,6 +13023,25 @@ intrinsics:
             - link: "llvm.aarch64.crc32cx"
               arch: aarch64,arm64ec
 
+  - name: "vabsd_s64"
+    doc: "Absolute Value (wrapping)."
+    arguments: ["a: {type[1]}"]
+    return_type: "{type[1]}"
+    attr:
+      - *neon-stable
+    assert_instr: [abs]
+    safety: safe
+    types:
+      - [i64, i64]
+    compose:
+      # FIXME(llvm): this should be `i64::wrapping_abs` but it optimizes differently.
+      # See https://github.com/llvm/llvm-project/issues/148388.
+      - LLVMLink:
+          name: "vabsd_s64"
+          links:
+             - link: "llvm.aarch64.neon.abs.i64"
+               arch: aarch64,arm64ec
```

Comment:
cc https://github.com/llvm/llvm-project/issues/148388

Just kept this as-is for now, but from what I can tell `i64::wrapping_abs` would be equivalent.

Reply:
The LLVM issue is closed, is this comment still relevant?

Reply:
I updated the comment. it's kind of subtle, but the intrinsic keeps the value in a simd register, while the scalar version uses GPRs.